### PR TITLE
add Modal styles

### LINKED ISSUE #126: Configure modal position/sizing

### PR DESCRIPTION
**Pull request by anserwaseem**
## Issue ticket number and link

Closes #126 

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [ ] I have added tests

### CODE REVIEW

File: packages/runtime/src/runtime/hooks/useNavigateModal.tsx [hunk 1]
```diff
@@ -18,7 +18,16 @@ export const useNavigateModalScreen: EnsembleActionHook<
...
+      : Boolean(action.maskClosable);
+  const position =
+    typeof action !== "string" ? action?.styles?.position : undefined;
+  const height =
+    typeof action !== "string" ? action?.styles?.height : undefined;
+  const width = typeof action !== "string" ? action?.styles?.width : undefined;
+  const margin =
+    typeof action !== "string" ? action?.styles?.margin : undefined;
+  const padding =
+    typeof action !== "string" ? action?.styles?.padding : undefined;
```
Comment (evshi) [inline packages/runtime/src/runtime/hooks/useNavigateModal.tsx:30]: Can make this check once and assign all at once with destructuring

File: packages/runtime/src/runtime/modal.tsx [hunk 2]
```diff
@@ -24,21 +31,61 @@ export const ModalWrapper: React.FC = () => {
...
+      setModalWidth(contentNode?.clientWidth || 0);
+      setModalHeight(contentNode?.clientHeight || 0);
+    }
+  }, [visible, isFullScreen, content]);
 
   const openModal = (
     content: React.ReactNode,
     newOptions: ModalProps,
   ): void => {
+    setKey((prevKey) => prevKey + 1);
```
Comment (evshi) [inline packages/runtime/src/runtime/modal.tsx:50]: What's the purpose of the key?

Comment (anserwaseem) [reply packages/runtime/src/runtime/modal.tsx:50]: when a new modal is opened from within an already opened one, Modal title was not updating, so putting a unique key was the solution. Although we use `Math.random()` as well, what do you say?

Comment (evshi) [reply packages/runtime/src/runtime/modal.tsx:50]: I see, so it's forcing a re-render here as I thought. I think this incrementing is fine

**Review by evshi — APPROVED** [review]

**Pull request merged**